package.json

```diff
@@ -110,5 +110,6 @@
     "testMatch": [
       "<rootDir>/test/**/*Test.ts"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha512.ff4579ab459bb25aa7c0ff75b62acebe576f6084b36aa842971cf250a5d8c6cd3bc9420b22ce63c7f93a0857bc6ef29291db39c3e7a23aab5adfd5a4dd6c5d71"
```

Comment:
Is this strictly necessary?  IMO it's better to be in its own release/PR, and should update to yarn >=2 anyhow (or at least the latest `1.22.x`).

Reply:
This is my package manage being a berk, it seems insistent on setting this everywhere. I need to go and disable it for now, because I think this needs an explicit PR.